weapons.yaml

```diff
@@ -1639,3 +1639,34 @@ NukePayload:
 		SmudgeType: LargeScorch
 		Size: 3
 		Delay: 15
+
+atomic:
```

Comment:
In fact nuke weapon and nuke reactor explodsion both uses NukePayload weapon. I recommend you to edit it. It also should polish nuke reactor explosion.